index.php

```diff
@@ -0,0 +1,45 @@
+<!-- WIREFRAME: https://balsamiq.cloud/s3isxla/p8k55mx/r2278 -->
+
+<?php
+
+require 'functions.php';
+
+$db = new PDO('mysql:host=db; dbname=collectorsApp', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `gwent_cards`;');
+$query->execute();
+
+$allCards = $query->fetchAll();
+
+/*uncomment if you want to check the whole db array
+print_r($allCards);
+*/
```

Comment:
i can see why this is handy but it shouldn't really be in the repo - commented out code in general is not the best to have in a repo

Reply:
removed